api/src/test/java/org/pmiops/workbench/api/UserMetricsControllerTest.java

```diff
@@ -261,6 +282,27 @@ public void testGetUserRecentResource() {
     assertEquals(recentResources.get(1).getCohort().getName(), "Cohort Name");
   }
 
+  @Test
+  public void testGetUserRecentResources_missingWorkspace() {
+    resource1.setWorkspaceId(9999); // missing workspace
+    when(mockUserRecentResourceService.findAllResourcesByUser(user.getUserId()))
+        .thenReturn(ImmutableList.of(resource1, resource2));
+    RecentResourceResponse recentResources =
+        userMetricsController.getUserRecentResources().getBody();
+    assertEquals(1, recentResources.size());
+    RecentResource foundResource = recentResources.get(0);
+    final Cohort cohort1 = foundResource.getCohort();
+    final Cohort cohort2 = cohortMapper.dbModelToClient(resource2.getCohort());
```

Comment:
If the `cohortMapper` is only used in test, I would bring it into the test package.

Reply:
I definitely intended it to be used in more places than the test.

Reply:
Moving it is no problem for now.